static/app/components/replays/replayView.tsx

```diff
@@ -72,19 +39,15 @@ const ReplayControllerWrapper = styled(PanelBody)`
 const PanelNoMargin = styled(Panel)<{isFullscreen: boolean}>`
   margin-bottom: 0;
   height: 100%;
-  ${p =>
-    p.isFullscreen
-      ? `
-      display: grid;
-      grid-template-rows: auto 1fr auto;
-      `
-      : ''}
+  display: grid;
+  grid-template-rows: auto 1fr auto;
 `;
 
 const PanelHeader = styled(_PanelHeader)<{noBorder?: boolean}>`
   display: block;
   padding: 0;
   ${p => (p.noBorder ? 'border-bottom: none;' : '')}
+  overflow: hidden;
```

Comment:
This is load-bearing! Very important!

This style ensures that this PanelHeader grows and shrinks based on it's parent, not the content inside. 

The content inside will be set to `height: 100%` and then there's some code in there to measure the size in pixels. If this was normal overflow then measured size would never shrink.

Reply:
If it's load 🐻 ing it should be in the code

Reply:
yeah, comments added